packages/x-adapter-platform/src/mappers/__tests__/response/related-tags-response.mapper.spec.ts

```diff
@@ -1,5 +1,5 @@
-import { PlatformRelatedTagsResponse } from '../../../types/response.types';
 import { relatedTagsResponseMapper } from '../../responses/related-tags-response.mapper';
+import { PlatformRelatedTagsResponse } from '../../../types/responses/related-tags-response.model';
 
 describe('related tags response mapper', () => {
   it('should map the related tags', () => {
```

Comment:
Could you remove this test file? Now it is in `mappers/responses/__tests__/related-tags-response.mapper.spec.ts`